packages/std/src/results/subcall.rs

```diff
@@ -45,6 +47,55 @@ where
     pub reply_on: ReplyOn,
 }
 
+/// This is used for cases when we use ReplyOn::Never and the id doesn't matter
+pub const UNUSED_MSG_ID: u64 = 123456789;
```

Comment:
```suggestion
/// This is used for cases when we use ReplyOn::Never and the ID doesn't matter
pub const UNUSED_MSG_ID: u64 = 0;
```

Save some space in JSON